app/controllers/events/routes.py

```diff
@@ -5,6 +5,28 @@
 from app.controllers.events import events_bp
 from app.logic.events import getEvents
 from app.logic.participants import sendUserData
+from datetime import datetime
+from app.models.eventRsvp import EventRsvp
+
+@events_bp.route('/events/<term>/', methods=['GET'])
```

Comment:
Isn't there already a route that does this?

Reply:
Yes. In main/routes.py there's a route that does the same thing and it's the one that we used for our navigation bar. So I deleted this one since we don't use it anywhere else.